src/boundingmesh/CMakeLists.txt

```diff
@@ -4,9 +4,16 @@ find_package(Eigen 3 REQUIRED)
 find_package(Coin)
 find_package(CGAL)
 find_package(GMP)
+find_package(assimp)
```

Comment:
Not sure if boundingmesh is used with Windows by anyone.
How likely will Windows users be able to get this compiled with this new dependency?

Reply:
Normally assimp should work fine with windows

Reply:
But I don't really know for the installation part